dashboard/Dashboard_SpreadSheetFriendly.md

```diff
@@ -1,69 +1,94 @@
-## Active HW TG Projects																																																						
-|	Name	|	TRL Target 	|	Status	|	TPL or PM	|	Repo	|	PC	|	PL	|	PA	|	PF	|	PF Estimate	|	Project Docs	|	RTD	|	Project Plan	|	Project Board	|																										
-|	--------------------	|	--------------------	|	--------------------	|	--------------------	|	--------------------	|	--------------------	|	--------------------	|	--------------------	|	--------------------	|	--------------------	|	--------------------	|	--------------------	|	--------------------	|	--------------------	|																										
-|	CORE-V-MCU	|	TRL 7 (CPU)	|	PA	|	[Greg Martin](https://github.com/gmartin102) <br /> +   [Hugh Pollitt-Smith](https://github.com/orgs/openhwgroup/people/hpollittsmith) 	|	[core-v-mcu](https://github.com/openhwgroup/core-v-mcu) 	|	20.10.05 (A)    	|	21.04.26 (A) 	|	21.12.12 (A)	|		|		|	[Project Launch](https://github.com/openhwgroup/programs/blob/master/Project-Descriptions-and-Plans/CORE-V-MCU-SoC/MCU-PL-Document.md)  <br /> +      [Plan Approved](https://github.com/openhwgroup/programs/blob/master/Project-Descriptions-and-Plans/CORE-V-MCU-SoC/PA-document-Oct-25-2021.md)	|	[CORE-V MCU User Manual](https://docs.openhwgroup.org/projects/core-v-mcu/)	|	[CORE-V MCU Project Libre Plan at PA](https://github.com/openhwgroup/programs/blob/master/Project-Descriptions-and-Plans/CORE-V-MCU-SoC/Core-V-MCU-SoC-Plan.pod)	|	[CV MCU Documentation Project Board](https://github.com/orgs/openhwgroup/projects/4)	|																										
-|	CORE-V MCU VERILATOR MODEL  	|		|	PL	|		|	[core-v-mcu](https://github.com/openhwgroup/core-v-mcu)   	|	21.03.22 (A)	|	21.05.24 (A) 	|		|		|		|	[Verilator Modeling PL](https://github.com/openhwgroup/programs/blob/master/Project-Descriptions-and-Plans/MCU-Verilator-Model/verilator-modeling-pl.md)	|		|		|		|																										
-|	CORE-V-MCU-DEVKIT	|	TRL 4	|	PL	|	[Joseph Julicher](https://github.com/n9wxu)	|	[core-v-mcu devkit](https://github.com/openhwgroup/core-v-mcu-devkit)	|	21.12.12 (A)	|	22.06.27 (A)	|		|		|		|	[Project Concept](https://github.com/openhwgroup/programs/blob/master/Project-Descriptions-and-Plans/CORE-V-MCU-DEV-KIT/CORE-V_DEV-KIT_project_concept_approved.md) <br /> + [Project Launch](https://docs.google.com/document/d/15v-1zsUEahpGljZNWOrmNEDTp1RlQCiLntFuJ-Hcw0I/edit)	|		|		|		|																										
-|	CORE-V-TRUSTED-MCU	|	TRL 7 (CPU)	|	PC	|	Abdoulaye Berthe	|		|	22.07.25 (A)	|		|		|		|		|	[CORE-V-MCU2 Project Concept](https://docs.google.com/document/d/1nW9rFAFHhNkNi53khY3hhynXaHIm4OqJ/edit)	|		|		|		|																										
+## Active HW TG Projects	
+																																																					
+|	Name				| Status	|	TPL or PM																																|	Repo																	|	PC		|	PL		|	PA		|	PF Estimate	|	Project Docs																																																																						|	RTD																				|	Project Plan			|	Project Board	|																										
```

Comment:
This table does not render properly on GitHub.

Reply:
Huh @MikeOpenHWGroup These render properly on a chrome markdown viewer. Not sure the issue...